Tests/ReplaySubjectTests.swift

```diff
@@ -0,0 +1,342 @@
+//
+//  ReplaySubjectTests.swift
+//  CombineExtTests
+//
+//  Created by Jasdev Singh on 4/13/20.
+//
+
+import Combine
+@testable import CombineExt
+import XCTest
+
+final class ReplaySubjectTests: XCTestCase {
+    private var cancellables = Set<AnyCancellable>()
+
+    private enum AnError: Error, Equatable {
+        case someError
+    }
+
+    func testReplaysNoValues() {
+        let subject = ReplaySubject<Int, Never>(bufferSize: 1)
+
+        var results = [Int]()
+
+        cancellables = Set([
+            subject
+                .sink(receiveValue: { results.append($0) })
+            ])
```

Comment:
Why not `.store(in: cancellables)` ? Would go this route just to use the "standard" way and not confuse anyone with the syntax

Reply:
Ah word, yea I can swap that in!

Reply:
(incoming in the next commit)